.changeset/purple-gifts-beam.md

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
just confirming: doesn't this need a changeset to trigger a new version?

Reply:
Yes, but there are already queued up version bumps to `@clerk/upgrade` so this will be part of that set of changes. We can make this an explicit changeset also if we feel that's useful

Reply:
nope, no need, totally cool with this being empty then!